include/secp256k1.h

```diff
@@ -318,8 +318,8 @@ SECP256K1_API SECP256K1_WARN_UNUSED_RESULT secp256k1_scratch_space* secp256k1_sc
 /** Destroy a secp256k1 scratch space.
  *
  *  The pointer may not be used afterwards.
- *  Args:       ctx: a secp256k1 context object.
- *          scratch: space to destroy
+ *  Args:       ctx: a secp256k1 context object (cannot be NULL)
```

Comment:
Well NULL is not a secp256k1, not sure if we need to say this everywhere.

But apparently we say this in some places, so we could make it consistent. But personally I'd prefer to omit it consistently.

Reply:
yeah I started by omitting it when it's obvious that it cannot be null, but then other places didn't omit so I settled on being explicit *everywhere*